docs/tasks/stuck_instance.md

```diff
@@ -0,0 +1,37 @@
+
+---
+title: Remove a Stuck Instance or Binding
+layout: docwithnav
+---
+
+For a variety of reasons, a service broker may stop responding to requests.
+A user may still need to remove instances and bindings belonging to that broker,
+bypassing the safeguards normally in place. Note that these are are destructive
+operations that may leave orphaned state still existing on the broker.
+
+Service Catalog maintains a [finalizer](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers)
+on each of the resources it maintains. This prevents the resource from being
+deleted before Service Catalog has a chance to reconcile the resource with
+the relevant service broker. In the event that an instance becomes stuck,
+manually removing the finalizer will allow the resource to be deleted. The finalizer can
+be manualy removed with `kubectl`, or by using the `--abandon` flag in the svcat `deprovision`
+and `unbind` commands.
+
+## Abandon an instance
+```console
+$ svcat deprovision foobar-mysql --abandon
+This action is not reversible and may cause you to be charged for the broker resources that are abandoned. If you have any bindings for this instance, please delete them manually with svcat unbind --abandon --name bindingName
+Are you sure? [y|n]: 
+y
+deleted foobar-mysql
+```
+Note that bindings belonging to the instance will not be cleaned up as part of the operation, and should be manually cleaned up before deprovisioning the instance.
+
+## Abandon a binding
+```console
+$ svcat unbind foobar-mysql --abandon
```

Comment:
nit:  this is actually abandoning all bindings for the specified instance (title says "Abandon a binding").  Maybe change the title or add additional example that shows `$ svcat unbind --name foobar-binding --abandon`

Reply:
good catch